From 27da30adddc6f9c5fcbc0d3a0b791db061d179cf Mon Sep 17 00:00:00 2001
From: OvchinnikovDmitrii <ovchinnikov.dmitrii@gmail.com>
Date: Fri, 11 Oct 2019 18:37:13 +0300
Subject: [PATCH] libavcodec/amfenc_h264.c: Changed loop filter flag default
 value.

The value has been changed to match the recommended(by AMF Encoder team)
---
 libavcodec/amfenc_h264.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libavcodec/amfenc_h264.c b/libavcodec/amfenc_h264.c
index 2c082e93bd9..7f2817f1157 100644
--- a/libavcodec/amfenc_h264.c
+++ b/libavcodec/amfenc_h264.c
@@ -366,6 +366,7 @@ static const AVCodecDefault defaults[] = {
     { "b",          "2M"  },
     { "g",          "250" },
     { "slices",     "1"   },
+    { "flags",      "+loop"},
     { NULL                },
 };
 
